deployments/boba.diamond.json

```diff
@@ -61,8 +61,11 @@
       "Receiver": "0x5439f8ca43f832DD21a28C5BF038dad4c07ad02c",
       "RelayerCelerIM": "",
       "ServiceFeeCollector": "0x894b3e1e30Be0727eb138d2cceb0A99d2Fc4C55D",
-      "LiFuelFeeCollector": "0xc4f7A34b8d283f66925eF0f5CCdFC2AF3030DeaE",
-      "TokenWrapper": "0x5215E9fd223BC909083fbdB2860213873046e45d"
+      "LiFuelFeeCollector": "0xc02FFcdD914DbA646704439c6090BAbaD521d04C",
+      "TokenWrapper": "0x5215E9fd223BC909083fbdB2860213873046e45d",
+      "GasRebateDistributor": "",
+      "LiFiDEXAggregator": "0x6140b987d6B51Fd75b66C3B07733Beb5167c42fc",
+      "ReceiverStargateV2": ""
```

Comment:
as pointed out, not deployed to all networks and therefore OK. I would expect you to not raise such comments in the future since the PR has nothing to do with the ReceiverStargateV2